tests/integration/components/new-project-modal/component-test.ts

```diff
@@ -14,7 +15,7 @@ const currentUserStub = Service.extend({
 
 module('Integration | Component | new-project-modal', hooks => {
     setupRenderingTest(hooks);
-
+    setupIntl(hooks, 'en-us');
```

Comment:
First you set up the hooks (the locale is optional and isn't needed here, only if you're verifying the locale specifically)

Reply:
We might want to put this into a custom `setupOSFRenderingTest`.